src/nupic/algorithms/SvmT.hpp

```diff
@@ -730,7 +730,7 @@ inline float svm<traits>::rbf_function(float* x, float* x_end, float* y) const
 {
   float sum = 0;
 
-#if defined(NTA_ASM) && defined(NTA_PLATFORM_win32) && defined(NTA_COMPILER_MSVC)
+#if defined(NTA_ASM) && defined(NTA_ARCH_32) && defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC)
```

Comment:
It's because this assembly code under this directive works only on Win32 and not Win64. By the way, I will remove 32 bits for Windows (and Darwin) systems as they are deprecated.